docs/reference-guides/interactivity-api/api-reference.md

```diff
@@ -1246,7 +1246,7 @@ store( 'mySliderPlugin', {
 				withScope( () => {
 					actions.nextImage();
 				} ),
-				3_000
+				3000
```

Comment:
This isn't a type. It's a number separator to improve readability. See: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Lexical_grammar#numeric_separators